core.py

```diff
@@ -80,16 +106,18 @@ def __init__(self, tof_lib):
         else:
             self.arduino = None
             self.PWMservo = PWM.Servo(pulse_incr_us=1)
-            for pin in range(0, 3):
-                i2c_lidar.xshut([LIDAR_PINS[pin]])
-                self.lidars.append(
-                    i2c_lidar.create(LIDAR_PINS[pin], tof_lib, 0x2a + pin)
-                )
+
+        # I2C lidar sensors should be enabled
+        # in both Arduino mode or direct pi mode.
+        for pin in range(0, 3):
+            i2c_lidar.xshut([LIDAR_PINS[pin]])
```

Comment:
This was the cause of the sensor issues last week; this should call i2c_lidar.xshut(LIDAR_PINS) before iterating over the pins turning them on and assigning addresses.